<launch>
    <node pkg="spacurs" type="teleop.py" name="teleop" output="screen">
        <param name="port" value="/dev/ttyUSB1"/>
        <param name="baud_rate" value="115200"/>
        <param name="remote_nodes" value="['1', '2']"/>
    </node>
</launch>